salt/modules/win_wua.py

```diff
@@ -169,39 +170,48 @@ def available(
             * Critical
             * Important
 
+        online (bool):
+            Tells the Windows Update Agent go online to update its local update
+            database. ``True`` will go online. ``False`` will use the local
+            update database as is. Default is ``False``
```

Comment:
You state the default is False, but the kwarg is `online=True,`

Reply:
Good catch!